modules/fts/pages/fts-search-response.adoc

```diff
@@ -1,14 +1,1316 @@
+  },
+  "sort": ["_id"],
+  "search_after": ["hotel_10180"],
+  "size": 3
+}
+----
+
+----
+{
+  "query": {
+      "match": "California",
+      "field": "state"
+  },
+  "sort": ["_id"],
+  "search_before": ["hotel_17595"],
+  "size": 4
+}
+----
+
+NOTE: A Full Text Search request that doesn't carry any pagination settings will return the first 10 results (`"size: 10", "from": 0`) ordered by _score_ sequentially from the highest to lowest.
+
+=== Pagination tips and recommendations
+
+The pagination of search results can be done using the 'from' and 'size' parameters in the search request. But as the search gets into deeper pages, it starts consuming more resources.  
+
+To safeguard against any arbitrary higher memory requirements, FTS provides a configurable limit bleveMaxResultWindow (10000 default) on the maximum allowable page offsets. However, bumping this limit to higher levels is not a scalable solution.
+
+To circumvent this problem, the concept of key set pagination in FTS, is introduced. 
+
+Instead of providing _from_ as a number of search results to skip, the user will provide the sort value of a previously seen search result (usually, the last result shown on the current page).  The idea is that to show the next page of the results, we just want the top N results of that sort after the last result from the previous page.
+
+This solution requires a few preconditions be met:
+
+* The search request must specify a sort order.
+
+NOTE: The sort order must impose a total order on the results.  Without this, any results which share the same sort value might be left out when handling the page navigation boundaries.  
+
+A common solution to this is to always include the document ID as the final sort criteria.                                       
+
+For example, if you want to sort by [“name”, “-age”], instead of sort by [“name”, “-age”, "_id”].
+
+
+With `search_after`/`search_before` paginations, the heap memory requirement of deeper page searches is made proportional to the requested page size alone. So it reduces the heap memory requirement of deeper page searches significantly down from the offset+from values.
+
+== Fields
+
+You can search multi-Collection indexes using the same old search requests. Since a multi-Collection index contains data from multiple source Collections, it’s useful to know the source Collection of their relevant hits.
+
+With multi-Collection indexes, each hit in the search results contains information about the Collection to which it belongs. This source Collection detail is available in the Fields section of each hit under the key _$c. See the image below for an example.
+
+image::fts-search-response-fields.png[,850,align=left]
+
+multi-Collection index search results in Couchbase 7.0
+
+You can also narrow your full-text search requests to only specific Collection(s) within the multi-Collection index. This focus speeds up your searches on a large index.
+
+== Sorting Query Results
+
+[abstract]
```

Comment:
Abstracts go underneath the first heading at the top of the document. Rule applies to all abstracts; just mentioning it here to save typing.

Reply:
And there should only be one abstract per document.

Reply:
updated